src/main/java/teammates/common/datatransfer/FeedbackResponseAttributes.java

```diff
@@ -51,6 +54,8 @@ public FeedbackResponseAttributes(String feedbackSessionName,
         this.recipientEmail = recipientEmail;
         this.recipientSection = recipientSection;
         this.responseMetaData = responseMetaData;
+        this.createdAt = new Date();
+        this.updatedAt = new Date();
```

Comment:
I believe the reason is it's that it needs to be set when the entity is created. Or are there any better (or correct) alternatives?